Project/Assets/Prairie/Framework/Script/Interaction/Annotation.cs

```diff
@@ -49,6 +49,8 @@ public class Annotation : Interaction
     private readonly float BOX_WIDTH = Screen.width / 2;
     private readonly float BOX_HEIGHT = Screen.height - 20;
 
+	private FirstPersonInteractor player;
+
```

Comment:
My mistake - turns out this property was already attached to `Interaction` (but for some reason git made it appear as an addition)